gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -578,6 +581,7 @@ public String getText(Object element) {
             switch (ttc.getType()) {
               case Graphics: return ttc.getApi();
               case Perfetto: return "System Profile";
+			  case Angle: return "OpenGL on ANGLE";
```

Comment:
Want a new pull-down option in "Type" for "OpenGL on ANGLE" but the trace itself is not really a new type. This selection really means "Do a Graphics Vulkan trace, but first run a couple of commands to make the OpenGL app run using ANGLE which will handle the Vulkan translation."

Reply:
We can start with having a new trace type and get the functional components working. Once the tracing is in a good state, we can think about refactoring this to be cleaner and integrated within the graphics trace type, if we want.